src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -45,8 +45,22 @@
     "max": None
 }
 
-scm_simulation_raw_data_validator = get_schema_validator("scm_simulation_raw_data_v1")
-simulation_raw_data_validator = get_schema_validator("simulation_raw_data_v1")
+
+class SimulationResultValidator:
```

Comment:
I do not understand the need for a separate class actually. Why not adding 
```python
self.results_validator = get_schema_validator("scm_simulation_raw_data")
```
into `CoefficientEndpointBuffer.__init__()`
and 
```python
self.results_validator = get_schema_validator("simulation_raw_data")
```
into `SimulationEndpointBuffer.__init__()`
?

Reply:
I did this for a couple of reasons:
1- There are more schema validators to be added in the future (for state variables, bids_offers_trades and configuration_tree). I felt that putting them all in the endpoint buffer will make the constructor of these classes even bigger.
2- I wanted to hide validation details from the endpoint buffers. This class does some sort of abstraction here.
3- (Personal pereference) I felt better to put all the real validation logic in one place to be able to manage them better in the future.

Reply:
Ok, after all this is only minor. Thanks for your explanation!